images/linux/Ubuntu2004-README.md

```diff
@@ -56,60 +55,60 @@
 ### Tools
 - Ansible 2.11.5
 - apt-fast 1.9.11
-- AzCopy 10.12.1 (available by `azcopy` and `azcopy10` aliases)
+- AzCopy 10.12.2 (available by `azcopy` and `azcopy10` aliases)
 - Bazel 4.2.1
 - Bazelisk 1.10.1
 - Bicep 0.4.613
 - Buildah 1.21.3 (apt source repository: https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable)
-- CMake 3.21.2
-- CodeQL Action Bundle 2.6.1
-- Docker Compose 1.29.2
+- CMake 3.21.3
+- CodeQL Action Bundle 2.6.2
+- Docker Compose 1.29.1
```

Comment:
It seems that the version was downgraded, need to generate a new image with 1.29.2